translations/jest.setup.js

```diff
@@ -0,0 +1,2 @@
+// @flow
```

Comment:
What is this file for?

Reply:
I added it to have exactly the same config in translations, api-client etc.